core/src/main/java/org/frankframework/util/XmlUtils.java

```diff
@@ -788,7 +789,7 @@ public static String createXPathEvaluatorSource(@Nullable String namespaceDefs,
 	 * @return An XSLT stylesheet generated to evaluate the XPath Expression
 	 */
 	@Nonnull
-	public static String createXPathEvaluatorSource(@Nonnull Function<String,String> xpathContainerSupplier, @Nonnull String xPathExpression, @Nonnull TransformerPool.OutputType outputMethod, boolean includeXmlDeclaration, @Nullable ParameterList params, boolean stripSpace, boolean ignoreNamespaces, int xsltVersion) {
+	public static String createXPathEvaluatorSource(@Nonnull UnaryOperator<String> xpathContainerSupplier, @Nonnull String xPathExpression, @Nonnull TransformerPool.OutputType outputMethod, boolean includeXmlDeclaration, @Nullable ParameterList params, boolean stripSpace, boolean ignoreNamespaces, int xsltVersion) {
```

Comment:
Sonar suggestion: when input and output type are the same, use the UnaryOperator